File: ezSCORES/ezSCORES.Model/Reviews.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ezSCORES.Model
{
	public class Reviews
	{
		public int Id { get; set; }

		public int UserId { get; set; }

		public int CompetitionId { get; set; }

		public float Rating { get; set; }
	}
}

File: ezSCORES/UI/ezscores_desktop/lib/models/reviews.dart
import 'package:json_annotation/json_annotation.dart';

part 'reviews.g.dart';

@JsonSerializable()
class Reviews {
	int? id;
	int? userId;
	int? competitionId;
	double? rating;

	Reviews();

	factory Reviews.fromJson(Map<String, dynamic> json) => _$ReviewsFromJson(json);

	Map<String, dynamic> toJson() => _$ReviewsToJson(this);
}
